pylint/extensions/docparams.py

```diff
@@ -202,25 +202,30 @@ class DocstringParameterChecker(BaseChecker):
     constructor_names = {"__init__", "__new__"}
     not_needed_param_in_docstring = {"self", "cls"}
 
-    def visit_functiondef(self, node):
-        """Called for function and method definitions (def).
-
-        :param node: Node for a function or method definition in the AST
-        :type node: :class:`astroid.scoped_nodes.Function`
+    def _skip_func_docstring(self, node):
+        """True either if the function `node` has a name matching the `no-docstring-rgx` pattern,
+        or a docstring shorter than `docstring-min-length`; False otherwise.
         """
-        node_doc = utils.docstringify(node.doc, self.config.default_docstring_type)
-
         # skip functions that match the 'no-docstring-rgx' config option
         no_docstring_rgx = get_global_option(self, "no-docstring-rgx")
         if no_docstring_rgx and re.match(no_docstring_rgx, node.name):
-            return
+            return True
 
         # skip functions smaller than 'docstring-min-length'
         lines = checker_utils.get_node_last_lineno(node) - node.lineno
         max_lines = get_global_option(self, "docstring-min-length")
-        if max_lines > -1 and lines < max_lines:
+        return max_lines > -1 and lines < max_lines
+
+    def visit_functiondef(self, node):
```

Comment:
We dropped python 3.5 some time ago ;)

```suggestion
    def visit_functiondef(self, node: astroid.scoped_nodes.Function) -> None:
```